src/rpc/client/ts/tests/specs/editing.spec.ts

```diff
@@ -134,7 +138,7 @@ describe('Editing', () => {
   describe('Delete', () => {
     it('Should delete some data', async () => {
       expect(0).to.equal(await getComputedFileSize(session_id))
-      const data = encode('abcdefghijklmnopqrstuvwxyz')
+      const data: Uint8Array = encode('abcdefghijklmnopqrstuvwxyz')
```

Comment:
Is there a reason why this `encode` function and the one from line 104 have different strings as an argument?

Reply:
There is a good reason.  They are different tests.